File: CursoJava/Exer/src/Outros/ListaAlgoritimo/Algoritimo/Exer06.java
/*
6. O restaurante a quilo Bem-Bão cobra R$12,00 por cada quilo de refeição.
Escreva um algoritmo que leia o peso do prato
montado pelo cliente (em quilos) e imprima o valor a pagar.
Assuma que a balança já desconte o peso do prato.
 */

package Outros.ListaAlgoritimo.Algoritimo;

import java.util.Scanner;

public class Exer06 {
    public static void main(String[] args) {
        double weight, price;
        Scanner sc = new Scanner(System.in);

        System.out.println("Qual o peso do prato montado? ");
        weight = sc.nextDouble();

        price = weight * 12.00;

        System.out.println("O valor a pagar será de R$" + price);

        sc.close();
    }
}

File: CursoWebCompleto/exer/listaAlgoritimos/algoritimo/exer06.js
/*
6. O restaurante a quilo Bem-Bão cobra R$12,00 por cada quilo de refeição. Escreva um algoritmo
que leia o peso do prato montado pelo cliente (em quilos) e imprima o valor a pagar. Assuma que a
balança já desconte o peso do prato.
 */

const price = 12;
const weight = 0.5;

const payment = price * weight;

console.log(`O valor a pagar é de R$${payment}.`);
